cmd/removeEnumerator.go

```diff
@@ -39,13 +39,15 @@ var NothingToRemoveError = errors.New("nothing found to remove")
 // provide an enumerator that lists a given resource (Blob, File)
 // and schedule delete transfers to remove them
 // TODO: Make this merge into the other copy refactor code
+// TODO: initEnumerator is significantly more verbose at this point, evaluate the impact of switching over
 func newRemoveEnumerator(cca *cookedCopyCmdArgs) (enumerator *copyEnumerator, err error) {
```

Comment:
yeah, there is quite a bit of uh, "weightiness" to initEnumerator, especially considering the individual little hacks to retain the same functionality as the older revisions of azcopy